client/app/components/BottomTabs.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import { StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+import { withNavigation } from 'react-navigation';
+
+import connectToRedux from '../redux/lib/connectToRedux';
+import sp from '../redux/lib/stateProperties';
+
+import styles, { colors } from '../config/styles';
+import SimpleLineIcons from '@expo/vector-icons/SimpleLineIcons';
+import MaterialCommunityIcons from '@expo/vector-icons/MaterialCommunityIcons';
+import MaterialIcons from '@expo/vector-icons/MaterialIcons';
+
+class BottomTabs extends Component {
+
+    constructor(props) {
+        super(props);
+        console.log(this.props.navigation)
+    }
+
+    render() {
+        return (
+            <View style={tabStyles.container}>
+                <View style={{
+                    ...styles.container.backgroundColorPrimary, 
+                    ...tabStyles.bottomTabs,
+                }}>
+                    <TouchableOpacity onPress={
+                        () => this.props.navigation.navigate('FriendsView')
+                    }>
+                        <SimpleLineIcons 
+                            name={'people'} 
+                            size={25} 
+                            color={colors.secondary} 
+                        />
+                    </TouchableOpacity>
+                    <TouchableOpacity onPress={
```

Comment:
I was having some trouble tapping the tab icons...maybe it might be worth upping the padding on the `TouchableOpacity`s?

Reply:
As in they're too small? And just the icons or the whole bottom tab bar? Or do ya mean that the tab bar and icons look big enough but that tapping on the bar but not exactly on the icon doesn't work?

Reply:
Ok here's what I did: I set the width of each item to be 33.3% so that if you like touch just to the right of the actual icon it will still work, so now the whole tab is touchable :) I can also make the icons themselves bigger etc. too if we want